src/app/components/admin-panel/admin-panel.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <app-sidenav>
```

Comment:
I had some specifics in mind when creating this component.  A reusable menu, where the plan is to try out an expandable menu, where the icons would always be showing.

Maybe a toolbar menu is able to do these things, but I did not expect them to. I'll take a second look at it.

Reply:
The idea was never to include this change for the toolbar. But possibly for other views in the main sections of the app.

Reply:
But the settings button is in the toolbar? Or is that only temporary?
I think I see what you are trying to do, and I'm not against it, it's just that the GUI got very messy now.

Reply:
The settings button is not intended to be related to the admin panel, it's for user settings. Hence, all users have it. However, I added it from the start just to have something in the toolbar. As we do not currently have any user settings, maybe it is time to remove it. I'm not sure how much user configuration would be possible in the final product. Possibly timezone, name change, email verification, who knows.

Technically, that button does not open a sidenav, but an administration view/module, that includes a side nav.

Reply:
Okay, that just made me more confused. You know you have two settings, right? One under the user that actually says Settings, and then this button. You can have it as is if you want to, I'm just confused thats all :P

Reply:
Oh, I see.

Settings is simply the name of the icon in the button for the administration panel, whilst settings is the name of the *function* that the user can take in their account menu. Would it make it better if I added these tool tips?

```typescript
  <button mat-button *ngIf="this.isAdmin" [routerLink]="'/admin'"
    matTooltip="Administration Panel">
    <mat-icon>settings</mat-icon>
  </button>

  <button mat-button *ngIf="this.isAuthenticated" matTooltip="User Account"
    [matMenuTriggerFor]="profileMenu">
    <mat-icon>account_circle</mat-icon>
  </button>
```

Reply:
Still just as confused, but tooltips are always good! 
I'm not confused THAT they are different but why, I would expect pretty much the same result when clicking on that icon as when clicking on Settings under profile

Reply:
What is it that makes you think they should have the same function do you think? In my opinion it seems pretty clear, and what I believe is a common approach to how to separate user functions from other navigation functions.

Reply:
Haha, you sound a bit like a psychologist here..
I think the main problem is that we look at user functions completely different, and maybe that you have a very clear picture of how everything looks when it's finished and I don't. I will agree to disagree on this and leave it like that, because I don't think we will get anywhere.